src/mozilla_sec_eia/models/sec10k/ex_21/create_labeled_dataset.py

```diff
@@ -157,6 +158,9 @@ def format_label_studio_output(
         with Path.open(json_file_path) as j:
             doc_dict = json.loads(j.read())
             filename = doc_dict["task"]["data"]["ocr"].split("/")[-1].split(".")[0]
+            # check if old local naming schema is being used
+            if len(filename.split("-")) == 6:
+                filename = "-".join(filename.split("-")[2:])
```

Comment:
I think there was a change in the way that file names were formatted, so this is a fix to make sure this function can handle the new and old filename format.

Reply:
Not a recommendation for this PR, but at some point I'd like to get the paths/filename handling ironed out and made more consistent throughout. It's gotten a bit janky, which I think makes sense when we've been rapidly prototyping and iterating in parallel, but would be nice to leave it a little more consistent so we don't hate ourselves in the future haha